langchain/src/memory/zep.ts

```diff
@@ -51,15 +55,17 @@ export class ZepMemory extends BaseChatMemory implements ZepMemoryInput {
     this.memoryKey = fields.memoryKey ?? this.memoryKey;
     this.baseURL = fields.baseURL;
     this.sessionId = fields.sessionId;
-    this.zepClient = new ZepClient(this.baseURL);
+    this.zepClient = new ZepClient(this.baseURL, fields.apiKey);
   }
 
   get memoryKeys() {
     return [this.memoryKey];
   }
 
   async loadMemoryVariables(values: InputValues): Promise<MemoryVariables> {
-    const lastN = values.lastN ?? 10;
+    // use either lastN provided by developer or undefined to use the
```

Comment:
Will this default to 10 for current users?

Reply:
This is a bug fix. The number of results returned should default to the message window size set on the server, not 10.